app/assets/javascripts/specifics.js.coffee

```diff
@@ -277,10 +277,29 @@ class hqmf.SpecificOccurrence
   
   # removes any rows that have the same value for OccurrenceA and OccurrenceB
   compactReusedEvents: ->
+    # LDY 8/25/17
+    # Something changed in the MAT so the "type" is no longer included in the HQMF. The backup
+    # type included too much detail (OccurrenceA_... and OccurrenceB_...), which made the types
+    # appear different for two occurrences of the same type.
+    # The code below ignores "Occ..." strings within the "type". This makes it so the type will
+    # now appear the same where appropriate.
+    # Note: OccurrenceA... is used for regular instances of an occurrence. OccA... is used for
+    # QDM variables.
+    typeLookup = {}
+    Object.keys(hqmf.SpecificsManager.typeLookup).map (type, index) =>
```

Comment:
@edeyoung would it make sense to move this normalization into the constructor for `hqmf.SpecificsManagerSingleton`? This function is the only one that calls `typeLookup`, and removing the `/Occ[a-zA-Z]*_/` substrings in the initialization loop saves you from having to do it every time `compactReusedEvents` is called. 

It also fits better, based on the `setup` and `test_specifics_initialized_properly` methods of `specifics_test.rb`, with what `type` should be (`Encounter`, vs. `OccA_Encounter`).

Reply:
if you still don't have to re-upload the measure and if everything continues to work as expected in the same way, then that makes sense to me.